package.json

```diff
@@ -18,7 +18,8 @@
     "@theia/plugin-packager": "latest",
     "rimraf": "2.6.2",
     "typescript-formatter": "7.2.2",
-    "typescript": "2.9.2"
+    "typescript": "2.9.2",
```

Comment:
is there a reason to not use same version of typescript of theia/che-theia ?

Reply:
set the version from `che-theia `